api/src/lib/token.js

```diff
@@ -0,0 +1,23 @@
+const jsonWebToken = require('jsonwebtoken')
+
+module.exports = class Token {
+  constructor (deps) {
+    this.config = deps(Config)
+    this.secret = this.config.data.getIn(['api', 'secret'])
+  }
+
+  get (prefix, duration) {
+    return jsonWebToken.sign({
+      prefix: prefix,
+      exp: String(Date.now() + duration)
```

Comment:
Date.now() returns milliseconds, but the property `exp` has to be expressed in seconds. `Date.now() / 1000` should do the trick.